cmd/runtimetest/main.go

```diff
@@ -642,6 +644,12 @@ func validate(context *cli.Context) error {
 			err := v.test(spec)
 			t.Ok(err == nil, v.description)
 			if err != nil {
+				if logLevel >= logrus.ErrorLevel {
+					// Skip 'SHOULD', 'MAY' ...
```

Comment:
I don't like overloading `logrus.ErrorLevel` to control this, since “SHOULD violations are fatal” is not the same as “I want to see debug logging”.  Can we add a setting to the `Validator` structure to control this for config validation, and a command-line parameter to `runtimetest` to control this for runtime validation?